static/js/src/advantage/react/components/Subscriptions/Notifications/Notifications.tsx

```diff
@@ -36,6 +41,16 @@ const Notifications = () => {
 
   return (
     <>
+      {hasPaymentMethod === false ? (
```

Comment:
Does the no payment method notification display on top of the other notifications?
for example, 
<img width="1285" alt="image" src="https://github.com/user-attachments/assets/2de76583-e4f7-44a0-a55b-348cc84907a5" />